src/main/java/reactor/netty/resources/MicrometerPooledConnectionProviderMeterRegistrar.java

```diff
@@ -29,13 +28,24 @@
 import static reactor.netty.Metrics.TOTAL_CONNECTIONS;
 
 /**
+ * Default implementation of {@link reactor.netty.resources.PooledConnectionProvider.MeterRegistrar}.
```

Comment:
{@link ConnectionProvider.MeterRegistrar} instead of {@link reactor.netty.resources.PooledConnectionProvider.MeterRegistrar}